docs/geps/gep-01.rst

```diff
@@ -0,0 +1,279 @@
+.. _gep-1:
+
+==========================
+GEP 1 — Naming Conventions
+==========================
+
+:Author: `Maximilian Blömer <https://github.com/mjbloemer>`_
+:Author: `Hans-Martin von Gaudecker <https://github.com/hmgaudecker>`_
+:Author: `Eric Sommer <https://github.com/Eric-Sommer>`_
+:Status: Draft
+:Type: Standards Track
+:Created: 2019-11-04
+:Resolution: <url> (required for Accepted | Rejected | Withdrawn)
+
+
+Abstract
+--------
+
+This GEP pins down naming conventions for GETTSIM — general rules for what data columns,
+parameters, Python identifiers (functions, variables), etc. should be called. In a
+nutshell and without explanations, these conventions are:
+
+* Names follow standard Python conventions (``lowercase_with_underscores``).
```

Comment:
If I'm not mistaken, Python conventions require constant values in `UPPERCASE`. In the context of a tax-ben system the parameters are constants. Should we write the param dictionary(ies) in upper case?

Reply:
This only applies to module-level constants.

E.g., after the import statements, define 
```python
PI = 3.14....
```
and never change it.

It does not apply to identifiers you would pass strings to functions etc.